README.md

```diff
@@ -242,6 +242,11 @@ The `th2_check1_active_tasks_number` metric separate rules with label `rule_type
 
 + Support for listening to messages using th2 transport protocol from a queue
 
+#### Changed:
+
++ Migrated to common `5.???`
++ Migrated to sailfish-utils `4.???`
```

Comment:
Why don't set the current versions from `gradle.properties` in those libs? We can forget to update them here if we update the actual dependencies. And we can forget to update `???` when we know the exact versions. So, why don't set the version that is most like will be here in the end right now